docs/_includes/ds4/install.html

```diff
@@ -68,10 +67,16 @@ <h3>Lasso</h3>
         <li>@ebay/skin/switch</li>
         <li>@ebay/skin/tab</li>
         <li>@ebay/skin/textbox</li>
+        <li>@ebay/skin/tooltip</li>
```

Comment:
Whoops.

Reply:
Yeah, we always miss these. I'll look into automating it as part of the new site.